src/chapters/thymeleaf-views/template-forms.rst

```diff
@@ -0,0 +1,168 @@
+Thymeleaf Forms
+================
+
+Another fine use for a template is to use it to build a generic form. Then you
```

Comment:
This sentence needs rephrasing. "We can also use templates to build generic forms!"

Reply:
Rephrased.  Thanks.